www/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@here/harp-www",
```

Comment:
perhaps harp-website would be a better name? Maybe more easy to remember?

Reply:
Applied.